pydis_site/templates/base/navbar.html

```diff
@@ -83,6 +84,25 @@
           <a class="navbar-item" href="{% url 'wiki:get' path="code-jams/" %}">
             Code Jams
           </a>
+          <hr class="navbar-divider">
+
+          {% if not user.is_authenticated %}
+            {% get_providers as socialaccount_providers %}
+
+            {% for provider in socialaccount_providers %}
+              {% if provider.id == "discord" %}
+                  <a class="navbar-item"
+                     href="{% provider_login_url provider.id process="login" scope=scope auth_params=auth_params %}"
+                  >Login with {{ provider.name }}</a>
+              {% endif %}
+            {% endfor %}
```

Comment:
why not just display the single login url instead of iterating over each until you find discord?

Reply:
As far as I can tell, there are a couple reasons to do this:

* There might be more than one Discord OAuth app to login with
* Discord OAuth might not be configured yet

Those aren't necessarily real concerns for the time being, I guess. I do still need to get the URL from the provider though, and this is the official way Allauth provides to do it.